src/models/ShowPreference.py

```diff
@@ -0,0 +1,133 @@
+from datetime import *
+
+from src.db.MyDBConnection import MyDBConnection
+
+from .Preference import Preference
+from .User import User
+from .Show import Show
+
+from datetime import date
+
+
+class ShowPreferences:
+    """
+        Deal with preference which is a join of a Preference and a Preference,
```

Comment:
Pas compris..

Reply:
Show preference is an object which allow us to put the join between the preference and the show, it's basically a show joined with it's "seen_flag"

Reply:
"a join of a Preference and a Preference"...